src/components/event.js

```diff
@@ -0,0 +1,52 @@
+//Individual Event contained within our Events page
+
+import React from 'react';
+
+
+//Exports out our individual event
```

Comment:
Before this gets reviewed further, I am going to add eslint to the project. This will help us have a more standardized format for code. Else I will making a lot more comments here than necessary. Instead eslint can guide us all to more consistent formatting.